README.md

```diff
@@ -1,105 +1,190 @@
+   for more information.
+2. Deploy APIClarity in K8s. It will be deployed in a new namespace `apiclarity`:
 
-```
-./deploy.sh <namespace1> <namespace2> ...
-```
+   ```shell
+   kubectl apply -f deployment/apiclarity.yaml`
+   ```
 
-Note:
-To build the WASM filter instead of using the pre-built binary, please follow the instructions on [wasm-filters](https://github.com/apiclarity/wasm-filters)
+3. Verify that APIClarity is running:
 
-5. Port forward to APIClarity UI:
-```
-kubectl port-forward -n apiclarity svc/apiclarity 9999:8080
-```
+   ```shell
+   $ kubectl get pods -n apiclarity
+   NAME                        READY   STATUS    RESTARTS   AGE
+   apiclarity-5df5fd6d98-h8v7t   1/1     Running   2          15m
+   mysql-6ffc46b7f-bggrv       1/1     Running   0          15m
+   ```
+
+4. Initialize and pull the `wasm-filters` submodule:
+
+   ```shell
+   git submodule init wasm-filters
+   git submodule update wasm-filters
+   cd wasm-filters
+   ```
+
+5. Deploy the Envoy Wasm filter for capturing the traffic:
+
+   Run the Wasm deployment script for selected namespaces to allow traffic
+   tracing.
+
+   The script will automatically:
+   - Deploy the Wasm filter binary as a config map
+   - Deploy the Istio Envoy filter to use the Wasm binary
+   - Patch all deployment annotations within the selected namespaces to mount
+     the Wasm binary
 
-6. Open APIClarity UI in the browser: [http://localhost:9999/](http://localhost:9999/)
+   ```shell
+   ./deploy.sh <namespace1> <namespace2> ...
+   ```
 
-7. Generate some traffic in the applications (e.g. [sock-shop demo](https://github.com/microservices-demo/microservices-demo)) in the traced namespaces and check APIClarity UI :)
+   **Note**:
+   To build the Wasm filter from source instead of using the pre-built binary,
+   please follow the instructions in the [wasm-filters](https://github.com/apiclarity/wasm-filters)
+   repository.
+6. Port forward to APIClarity UI:
 
+   ```shell
+   kubectl port-forward -n apiclarity svc/apiclarity 9999:8080
+   ```
+
+7. Open APIClarity UI in the browser: <http://localhost:9999/>
+8. Generate some traffic in the applications in the traced namespaces and check
+   the APIClarity UI :)
+
+## Testing with a demo application
+
+A good demo application to try APIClarity with is the [Sock Shop Demo](https://microservices-demo.github.io/).
+
+To deploy the Sock Shop Demo follow these steps:
+
+1. Clone the Sock Shop Demo repository:
+
+   ```shell
+   git clone https://github.com/microservices-demo/microservices-demo.git
+   ```
+
+2. Create the `sock-shop` namespace and enable Istio injection:
+
+   ```shell
+   kubectl create namespace sock-shop
+   kubectl label namespaces sock-shop istio-injection=enabled
+   ```
+
+3. From the `microservices-demo` git repository cloned earlier deploy the demo
+   to your cluster:
+
+   ```shell
+   cd microservices-demo
+   kubectl apply -f deploy/kubernetes/complete-demo.yaml
+   ```
```

Comment:
We can just do
1. Create the `sock-shop` ...
2. Deploy the demo... `kubectl apply -f https://raw.githubusercontent.com/microservices-demo/microservices-demo/master/deploy/kubernetes/complete-demo.yaml`

WDYT?

Reply:
Great idea! Added the suggestion and rebased onto master.